universal-login-relayer/lib/services/messages/PendingMessagesStore.ts

```diff
@@ -56,6 +56,10 @@ export default class PendingMessagesStore implements IPendingMessagesStore {
     };
   }
 
+  getCollectedSignatures(messageHash: string) {
+    return this.messages[messageHash].collectedSignatures;
```

Comment:
rename
- `getCollectedSignatures` to `getCollectedSignatureKeyPairs`
- `collectedSignatures` to `collectedSignatureKeyPairs`